odoo/signals.py

```
@@ -0,0 +1,123 @@
+from care.users.models import User
+from odoo.resource.account_move.delivery_order import OdooDeliveryOrderResource
+from odoo.resource.account_move.invoice import OdooInvoiceResource
+from odoo.resource.account_move_payment.payment import OdooPaymentResource
+from odoo.resource.product_category.category import OdooCategoryResource
+from odoo.resource.product_product.resource import OdooProductProductResource
+from odoo.resource.res_partner.resource import OdooPartnerResource
+from odoo.resource.res_user.resource import OdooUserResource
+
+
+@receiver(post_save, sender=User)
+def sync_user_to_odoo(sender, instance, created, **kwargs):
```

Ruff (0.14.3)

Unused function argument: `sender`

(ARG001)